### PR TITLE
Fix issues in solr directory

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

**Review by sashano69 — APPROVED** [review]

File: reference/solr/solrquery/addgroupfield.xml [hunk 1]
```diff
@@ -29,12 +29,20 @@
...
      </para>
     </listitem>
    </varlistentry>
   </variablelist>
  </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Returns an <classname>SolrQuery</classname> instance.
```
Comment (Girgias) [inline reference/solr/solrquery/addgroupfield.xml:42]:
```suggestion
   Returns an instance of <classname>SolrQuery</classname>.
```

File: reference/solr/solrquery/getexpand.xml [hunk 2]
```diff
@@ -18,6 +18,18 @@
...
 
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Returns &true; if group expanding is enabled.
```
Comment (Girgias) [inline reference/solr/solrquery/getexpand.xml:29]: False otherwise?

File: reference/solr/solrquery/setgroupmain.xml [hunk 3]
```diff
@@ -36,16 +36,10 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   
+   Returns an <classname>SolrQuery</classname> instance.
```
Comment (Girgias) [inline reference/solr/solrquery/setgroupmain.xml:39]:
```suggestion
   Returns an instance of <classname>SolrQuery</classname>.
```

File: reference/solr/solrquery/sethighlightformatter.xml [hunk 4]
```diff
@@ -42,6 +42,13 @@
    </variablelist>
   </para>
  </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Returns an <classname>SolrQuery</classname> instance.
```
Comment (Girgias) [inline reference/solr/solrquery/sethighlightformatter.xml:49]:
```suggestion
   Returns an instance of <classname>SolrQuery</classname>.
```

File: reference/solr/solrquery/sethighlightsimplepost.xml [hunk 5]
```diff
@@ -45,6 +45,13 @@
   </para>
  </refsect1>
 
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Returns an <classname>SolrQuery</classname> instance.
```
Comment (Girgias) [inline reference/solr/solrquery/sethighlightsimplepost.xml:51]:
```suggestion
   Returns an instance of <classname>SolrQuery</classname>.
```

**Review by Girgias — CHANGES_REQUESTED** [review]

File: reference/solr/solrquery/getexpand.xml [hunk 6]
```diff
@@ -18,6 +18,18 @@
...
 
+ <refsect1 role="parameters">
+  &reftitle.parameters;
+  &no.function.parameters;
+ </refsect1>
+
+ <refsect1 role="returnvalues">
+  &reftitle.returnvalues;
+  <para>
+   Returns &true; if group expanding is enabled,&return.falseforfailure;.
```
Comment (cmb69) [inline reference/solr/solrquery/getexpand.xml:29]: I *guess* that should be
```suggestion
   Returns whether group expanding is enabled.
```

**Review by cmb69 — APPROVED** [review]
Thank you! Maybe change the commented return value section?

File: reference/solr/solrquery/getexpand.xml (hunk 6 above)
Comment (saundefined) [reply reference/solr/solrquery/getexpand.xml]: Thank you! Looks good =)

**Review by Girgias — APPROVED** [review]
LGTM

**Pull request merged**